allauth/socialaccount/providers/apple/client.py

```diff
@@ -52,7 +52,7 @@ def generate_client_secret(self):
         return client_secret
 
     @classmethod
-    def is_json_response(cls, response):
```

Comment:
I've already updated that [here](https://github.com/pennersr/django-allauth/pull/2424/commits/00b5134c6206249bbadf82a6a1012eba019eb43a)